### PR TITLE
Concentrate multiple entries into FOR Loop

### PR DESCRIPTION
**Pull request by Ruined1**
Concentrate multiple entries for Windows Defender and WifiSense into FOR loop read from single dictionary.

### CODE REVIEW

**Comment by 10se1ucgo** [comment]
Dictionaries can't have multiple values unless you use a list. For example,
```python
dictionary = {'Delivery Optimization Download': r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config': 'DODownloadMode'
```
should be
```python
dictionary = {'Delivery Optimization Download': [r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config', 'DODownloadMode']
```

And then in the for loop you would do

```python
for title, registry in dictionary.viewitems():
 print name  # Should output "Delivery Optimization Download"
 print registry[0]  # Should output r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config'
 print registry[1]  # Should output 'DODownloadMode'


Lastly, as opposed to using if undo and if not undo, because every value is the same (0 for disable, 1 for enable) just make your function take an input of `dwordval`, and for undo, pass on the value 1 (not a string) and for not undo pass on a value of 0 (not a string). For example, this was done in the service regkey, so you can take a look at how it was done there.

**Comment by W4RH4WK** [comment]
Is there a special reason why you'd use `viewitems()` instead of `iteritems()` to iterate over the dict?

**Comment by 10se1ucgo** [comment]
IIRC In Python 3.x `iteritems()` is replaced by `items()`, and `viewitems()` is Python 2.x's version of `items()`

**Comment by W4RH4WK** [comment]
The quote underlines my question i think, why would i create a whole list in the first place when you can do with a generator which gives you the item you need on the fly. It's more efficient with the generator.

**Comment by 10se1ucgo** [comment]
Considering that it's removed in 3.x, I think there is a good reason for it.

**Comment by W4RH4WK** [comment]
Yeah just found it. `iteritems()` has been replaced with `items()` in Python3, since this is even more efficient it has been backported to Python2 as `viewitems()`. So in Python2: `items() < iteritems() < viewitems()`

**Comment by Ruined1** [comment]
Fantastic! Remember this is only my second or third day working with python, so you have no idea how much I appreciate the help :sweat_smile: I'll fix the errors and make the suggested improvement!

**Comment by Ruined1** [comment]
That is fixed, for my next piece of optimization, I'm thinking of concentrating the code as a whole. Make all registry-related changes create a standardized dictionary (like I've done with this function) and then create a dedicated function that takes those dictionaries as an argument and passes them through a single for loop.

File: run.py [hunk 1]
```diff
@@ -439,107 +439,28 @@ def modifyserviceregs(dwordval):
...
-            _winreg.CloseKey(wdsamplekey)
-            print "Windows Defender Sample Submission key successfully modified."
-        except (WindowsError, IOError):
-            print "Unable to modify Windows Defender Sample Submission key. Deleted, or is the program not elevated?"
-
+def stopdefendwifi(regdwordval):
+  
+    # Windows Defender and WifiSense keys
+    stopdefendwifidict = {'Delivery Optimization Download': [r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config', 'DODownloadMode'],
+                        'WifiSense Credential Share': [r'SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features': 'WiFiSenseCredShared'\,
```
Comment (10se1ucgo) [inline run.py]: Typo, that backslash at the end should be a square bracket

Comment (10se1ucgo) [inline run.py]: Everything below here: Replace the : with commas inside of the list

File: run.py [hunk 2]
```diff
@@ -439,107 +439,28 @@ def modifyserviceregs(dwordval):
...
-            _winreg.SetValueEx(wdsamplekey, "SubmitSamplesConsent", 0, _winreg.REG_DWORD, 1)
-            _winreg.CloseKey(wdsamplekey)
-            print "Windows Defender Sample Submission key successfully modified."
-        except (WindowsError, IOError):
-            print "Unable to modify Windows Defender Sample Submission key. Deleted, or is the program not elevated?"
-
+def stopdefendwifi(regdwordval):
+  
+    # Windows Defender and WifiSense keys
+    stopdefendwifidict = {'Delivery Optimization Download': [r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config', 'DODownloadMode'],
```
Comment (10se1ucgo) [inline run.py]: Everything below here: Replace the : with commas inside of the list

**Comment by Ruined1** [comment]
Done.

**Comment by 10se1ucgo** [comment]
In the lists inside of the dictionaries you have colons ('blahblah': 'blahblah') instead of commas ['blahblah', 'blahblah']

**Comment by Ruined1** [comment]
Done. I need to get an IDE -_- I feel bad having you stumble on my errors for me.

**Comment by 10se1ucgo** [comment]
Don't, if anything I'm glad I'm doing this. Plus, you're doing work for me. Thanks :)

**Pull request merged**